client/src/components/Picture/UploadPic.js

```diff
@@ -0,0 +1,211 @@
+import React, { useRef, useState } from "react";
+import { Modal } from "antd";
+import { CameraOutlined } from "@ant-design/icons";
+import axios from "axios";
+import BaseButton from "../Button/BaseButton";
+import ReactCrop from "react-image-crop";
+import styled from "styled-components";
+import { theme } from "../../constants/theme";
+
+const { colors } = theme;
+
+const CustomSubmitButton = styled(BaseButton)`
+  background: ${colors.royalBlue};
+  color: ${colors.white};
+`;
+
+const CustomCancelButton = styled(BaseButton)``;
+
+function isImageFile(file) {
+  return file && file["type"].split("/")[0] === "image";
+}
+
+const UploadPic = ({ cameraIconSize, user }) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  const [photoURL, setPhotoURL] = useState();
+  const [uploadError, setUploadError] = useState();
+  const [image, setImage] = useState();
+  const [crop, setCrop] = useState({
+    aspect: 1 / 1,
+    unit: "px",
+    height: 250,
+    width: 250,
+  });
+  const imgUpload = useRef(null);
+
+  const imageLoaded = (image) => {
+    setImage(image);
+  };
+
+  const handleImage = (e) => {
+    setUploadError("");
+    const file = e.target.files[0];
+    if (file && file.size > 5001520) {
+      setUploadError("Please upload an image of size less than 5 MB.");
+    } else if (!isImageFile(file)) {
```

Comment:
From a UX standpoint, it probably makes more sense to check if the file is an image file first before checking its file size, otherwise a user may try to upload a 10 MB PDF file but would get the large file size error message first rather than the non-image file error message.